requirements.txt

```diff
@@ -1,2 +1,2 @@
-nypl-py-utils==0.0.6
-pandas
\ No newline at end of file
+nypl-py-utils==1.0.0
```

Comment:
Will this install all dependencies of py-utils, and does that possibly exceed the lambda storage allowance?

Reply:
It actually installs none of the dependencies -- just the package itself